snp_measurement/src/stage0.rs

```diff
@@ -34,6 +40,44 @@ const LEGACY_TOP: PhysAddr = PhysAddr::new(0x10_0000);
 /// The maximum size of the shadow firmware for legacy boot.
 const LEGACY_MAX_SIZE: usize = 128 * 1024;
 
+/// The reverse offset from the end of the firmware blob to the end of the GUID tables.
+const GUID_TABLE_END_OFFSET: usize = 0x20;
+
+/// The size of the header of an entry in the GUID table.
+///
+/// The header consists of a 16 byte GUID and a 16-bit length field.
+const GUID_TABLE_ENTRY_HEADER_SIZE: usize = size_of::<u128>() + size_of::<u16>();
```

Comment:
Minor nit, but it is it really a header if it is at the end? :)

Reply:
I don't think it is great, but couldn't think of a better name. I am open to suggestions.